docs/docs/manual/cellediting.md

```diff
@@ -0,0 +1,162 @@
+
+## Transform
+
+Select “Edit cells” > “Transforms” to open up an expressions window. From here, you can apply [expressions](expressions) to your data. The simplest examples are GREL functions such as `toUppercase` or `toLowercase`, used in expressions as `toUppercase(value)` or `toLowercase(value)`. In all of these cases, `value` is the value in each cell in the selected column. 
+
+Use the preview to ensure your data is being transformed correctly. 
+
+You can also switch to the “History” tab inside the expressions window to reuse expressions you’ve already attempted in this project, whether they have been undone or not. 
+
+OpenRefine offers you some frequently-used transformations in the next menu option, “Common transforms.” For more custom transforms, read up on [expressions](expressions). 
+
+## Common transforms
+
+### Trim leading and trailing whitespace
+
+Often cell contents that should be identical, and look identical, are different because of space or line-break characters that are invisible to users. This function will get rid of any characters that sit before or after visible text characters.
+
+### Collapse consecutive whitespace
+
+You may also find that some text cells contain what look like spaces but are actually tabs, or contain multiple spaces in a row. This function will remove all space characters that sit in sequence and replace them with a single space. 
+
+### Unescape HTML 
+
+Your data may come from an HTML-formatted source that expresses some characters through references (such as "&amp;nbsp;" for a space, or "%u0107" for a ć) instead of the actual Unicode characters. You can use the “unescape HTML entities” transform to look for these codes and replace them with the characters they represent. 
+
+### Replace smart quotes with ASCII
+
+Smart quotes (or curly quotes) recognize whether they come at the beginning or end of a string, and will generate an “open” quote (“) and a “close” quote (”). These characters are not ASCII-compliant (though they are UTF8-compliant) so you can use this tranform to replace them with a straight double quote character (") instead. 
+
+### Case transforms
+
+You can transform an entire column of text into UPPERCASE, lowercase, or Title Case using these three options. This can be useful if you are planning to do textual analysis and wish to avoid case-sensitivity (which many functions are) causing problems in your analysis. 
+
+### Data-type transforms
+
+As detailed in [Data types](exploring#data-types), OpenRefine recognizes seven data types: string, number, boolean, date, array, error, and null. When you use these transforms, OpenRefine will check to see if the given values can be converted, then both transform the data in the cells (such as “3” as a text string to “3” as a number) and convert the data type on each successfully transformed cell. Cells that cannot be transformed will output the original value and maintain their original data type. 
```

Comment:
```suggestion
As detailed in [Data types](exploring#data-types), OpenRefine recognizes five data types: string, number, boolean, date, and array. It also has an internal data type for storing errors. When you use these transforms, OpenRefine will check to see if the given values can be converted, then both transform the data in the cells (such as “3” as a text string to “3” as a number) and convert the data type on each successfully transformed cell. Cells that cannot be transformed will keep the original value and data type. 
```

Reply:
Apparently we don't want to include "array" in this list either, according to last week's conversation about data types. This whole sentence will be cleaned up.